react/features/notifications/middleware.ts

```diff
@@ -132,8 +134,8 @@ MiddlewareRegistry.register(store => next => action => {
         // Do not display notifications for the virtual screenshare and whiteboard tiles.
         if (conference
             && !p.local
-            && !p.isVirtualScreenshareParticipant
-            && !p.isWhiteboard
+            && !isScreenShareParticipant(p)
+            && p.fakeParticipant !== FakeParticipant.Whiteboard
```

Comment:
maybe it would be useful to add a similar selector for whiteboard as the one for screen share